### DISCUSSION TITLE
Caldera Issue

### DISCUSSION THREAD

**Discussion by Rhymtester** [mitre/caldera#2618] [General]
We are running Caldera (4.x) for Pentesing, We are able to create an operation sucessfully but the operation is not going to complete, it is always showing running state only instead of finished. But after degrade version (3.x), it is completing for some adversary profiles.

**Comment by clenk** [comment]
Hi @Rhymtester. Are the abilities in the operation completing, or are they not running either? Also, have you set the "Auto-close" option on the operation?

![image](https://user-images.githubusercontent.com/402940/177349169-60ed25bf-1284-4b86-b1ab-4e487a82ef75.png)

**Reply by Rhymtester** [reply]
Hi Clenk, Thanks for your response, the operation status showing always in running state (halting), yes even for autoclose option. Is there any issue with the version?

**Reply by clenk** [reply]
@Rhymtester sorry, we haven't been able to replicate the issue on our end. Are there any errors in the browser's console, or in the server's logs (try running the server with `--log DEBUG`)?

**Reply by Rhymtester** [reply]
Ok, thanks Clenk
